### PR TITLE
Added braces to C

### PR DESCRIPTION
**Pull request by radarhere**
See #4493

### CODE REVIEW

File: src/libImaging/GetBBox.c [hunk 1]
```diff
@@ -94,21 +96,24 @@ ImagingGetProjection(Imaging im, UINT8* xproj, UINT8* yproj)
...
                     xproj[x] = 1;\
                 }\
-                if (has_data)\
-                    yproj[y] = 1;\
-                }
+            }\
+            if (has_data) {\
+                yproj[y] = 1;\
+            }\
+        }
```
Comment (hugovk) [inline src/libImaging/GetBBox.c:108]: Just a note.

Before:
Visually, both `if` blocks are indented as if they are both in the `for` loop.

But it wouldn't make sense for both to be in the `for` loop, so it was functionally correct before, and the change makes it clearer.

File: src/libImaging/JpegEncode.c [hunk 2]
```diff
@@ -261,17 +264,19 @@ ImagingJpegEncode(Imaging im, ImagingCodecState state, UINT8* buf, int bytes)
...
                 context->extra_offset += n;
-                if (context->extra_offset >= context->extra_size)
+                if (context->extra_offset >= context->extra_size) {
                     state->state++;
-                else
+                } else {
                     break;
+                }
             } else
                 state->state++;
```
Comment (hugovk) [inline src/libImaging/JpegEncode.c:281]:
```suggestion
            } else {
                state->state++;
            }
```

**Comment by hugovk** [comment]
A few other `else`s:

```console
grep -r " else$" . --include "*.c"
./src/libImaging/BoxBlur.c:    else
./src/libImaging/BoxBlur.c:    else
./src/libImaging/BoxBlur.c:    else
./src/libImaging/RawEncode.c:        } else
./src/libImaging/JpegEncode.c:            } else
./src/libImaging/XbmEncode.c:                } else
./src/libImaging/GifDecode.c:    } else
./src/_webp.c:        } else
./src/_webp.c:        } else
```

The `if`s are harder to `grep`, this finds some plus many false positives:

`grep -r " if.*[^{]$" . --include "*.c"`

**Comment by radarhere** [comment]
Ok, I've done a pass over the code using your `grep` commands, and pushed the commit.

**Pull request merged**